### ISSUE TITLE
Build fails on Ubuntu 21.10

### ISSUE THREAD

**Issue by kkbodke** [vmware/esx-boot#9]
### Describe the bug

make fails on Ubuntu 21.10.

/usr/bin/python does not come by default on Ubuntu 21.10

$ make VERBOSE=2 
gcc -m32 -march=i386 -mpreferred-stack-boundary=2 -msoft-float -fPIE -fno-asynchronous-unwind-tables -D__COM32__ -msoft-float --sysroot='/' -Donly_ia32 -Donly_x86 -DVMWARE_EDK2_CHANGES -ffreestanding -fno-exceptions -Os -fstrength-reduce -ffast-math -fomit-frame-pointer -finline-limit=2000 -freg-struct-return -fno-strict-aliasing -falign-functions=0 -falign-jumps=0 -falign-labels=0 -falign-loops=0 -fwrapv -fvisibility=hidden -fstack-protector -fstack-protector-all -mstack-protector-guard=global -W -Wall -Werror -std=c99 -Wwrite-strings -Wstrict-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wvla -Woverlength-strings -Wredundant-decls -Wformat -Wno-implicit-fallthrough -g  -I../libc/include -I../include -I../include/x86 -I../libmd5 -I. -fno-stack-protector -c -o ../build/com32/mboot/trampoline.o trampoline.c
[com32 GETKEYS] mboot/signinfo.c
LD_LIBRARY_PATH=: \
../env/getkeys.py < signinfo.json > ../build/com32/mboot/signinfo.c
/bin/sh: 1: ../env/getkeys.py: not found
make[2]: *** [rules.mk:31: ../build/com32/mboot/signinfo.c] Error 127
make[2]: Leaving directory '/home/kkbodke/esx-boot/test/esx-boot/mboot'
make[1]: *** [Makefile.main:37: mboot] Error 2
make[1]: Leaving directory '/home/kkbodke/esx-boot/test/esx-boot'
make: *** [Makefile:25: com32] Error 2

### Reproduction steps

```bash
1.Run make on Ubuntu 21.10 to reproduce the issue on the release main branch.
2.
3.
...
```

### Expected behavior

Build should be successful

**Comment by TimothyPMann** [comment]
This is not exactly a bug, as no one promised that you can build this package with only the packages that come by default on Ubuntu 21.10.

However, I will update the script to work with python3, as we don't really want to continue to have python2 dependencies.

**Comment by TimothyPMann** [comment]
Fixed by commit bb3c96c4d852f993e355253db9eb249ba319b9f6 -- now python2 or python3 both work.
